src/Resources/translations/messages.fr.yml

```diff
@@ -11,7 +12,18 @@ sylius_invoicing_plugin:
         issued_for_order: 'Émise pour la commande'
         issued_from: 'Émise de'
         manage_invoices: 'Gérez vos factures'
+        net_total: 'Total TTC'
+        net_value: 'Prix TTC'
+        no: 'N°'
         order_number: 'Numéro de commande'
+        payment:
+            paid: 'Payée'
+            yes: 'Oui'
+            no: 'Non'
         resend_invoice: 'Réenvoyer'
         seller: 'Vendeur'
+        tax_amount: 'Montant TVA'
```

Comment:
IMO it should rather be `Montant taxes` because taxes can be something else than VAT only.

And then, if your shop contains only VAT as tax, you can override the translation locally

Reply:
I was asking myself if it is better to write `TVA` or `Taxe`, but I think you are right.